tutorials/XGI in 5 minutes.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>There are more than 2 options: you can also read from file or load a dataset. I would just say "in XGI in 1 min we saw option 1 (describe), now we're using a second option (describe)"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>